Traceback (most recent call last):
  File "D:\MCTS-AHD-master\problems\tsp_constructive\eval.py", line 11, in <module>
    from gpt import select_next_node_v2 as select_next_node
  File "D:\MCTS-AHD-master\problems\tsp_constructive\gpt.py", line 1
    importance to the distance to the current node early in the tour and more to the product later, creating a different balance between proximity and long-term savings.}
                                                                                                                                                                         ^
SyntaxError: unmatched '}'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\MCTS-AHD-master\problems\tsp_constructive\eval.py", line 13, in <module>
    from gpt import select_next_node
  File "D:\MCTS-AHD-master\problems\tsp_constructive\gpt.py", line 1
    importance to the distance to the current node early in the tour and more to the product later, creating a different balance between proximity and long-term savings.}
                                                                                                                                                                         ^
SyntaxError: unmatched '}'
